core/src/main/java/org/apache/calcite/rel/logical/LogicalWindow.java

```diff
@@ -66,13 +67,13 @@ public final class LogicalWindow extends Window {
    *
    * <p>Use {@link #create} unless you know what you're doing.
    *
-   * @param cluster Cluster
-   * @param traitSet Trait set
-   * @param hints   Hints for this node
-   * @param input   Input relational expression
+   * @param cluster   Cluster
```

Comment:
In the codebase we generally don't format javadoc as you do here, can you please revert these changes?